spring-cloud-kubernetes-client-discovery/src/main/java/org/springframework/cloud/kubernetes/client/discovery/reactive/KubernetesInformerReactiveDiscoveryClientAutoConfiguration.java

```diff
@@ -88,24 +85,4 @@ public KubernetesInformerReactiveDiscoveryClient kubernetesReactiveDiscoveryClie
 				serviceLister, endpointsLister, serviceInformer, endpointsInformer, properties);
 	}
 
-	@Bean
-	@ConditionalOnMissingBean
-	public CatalogSharedInformerFactory catalogSharedInformerFactory() {
```

Comment:
both `CatalogSharedInformerFactory` and `SpringCloudKubernetesInformerFactoryProcessor` are removed, as we do not use them anymore. They were using deprecated functionality anyway